Examples/Io/Root/src/RootSpacepointWriter.cpp

```diff
@@ -128,6 +129,8 @@ ActsExamples::ProcessCode ActsExamples::RootSpacepointWriter::writeT(
     m_y = sp.y() / Acts::UnitConstants::mm;
     m_z = sp.z() / Acts::UnitConstants::mm;
     m_r = sp.r() / Acts::UnitConstants::mm;
+    m_t = sp.t().value_or(std::numeric_limits<double>::quiet_NaN()) /
+          Acts::UnitConstants::ns;
```

Comment:
I guess this would throw an FPE in case of `NaN`?
```suggestion
    m_t = sp.t().has_value() ? sp.t().value() / Acts::UnitConstants::ns : std::numeric_limits<double>::quiet_NaN();
```